data_registry/models.py

```diff
@@ -68,6 +68,12 @@ class Status(models.TextChoices):
         "<dd>The name of the dataset in Pelican</dd>"
         "</dl>",
     )
+    process_notes = models.JSONField(
+        blank=True,
+        default=dict,
+        help_text="The collection notes from Kingfisher Process, with every collection note level code as key, with an"
+        "array of a two-value array with the collection note description as string and data as dict",
```

Comment:
This text appears in the admin, so we can keep it short. You'll need to edit the migration.

```suggestion
        help_text="The collection notes from Kingfisher Process.",
```